modules/cij/runner.py

```diff
@@ -814,6 +818,7 @@ def main(args, conf):
         trun = trun_setup(args, conf)   # Construct 'trun' from args and conf
     except CIJError as ex:
         cij.err("main:FAILED to start testrun: %s" % ex)
+        return 1
```

Comment:
I have added this return value when we catch an exception.
It is strange to me that it was not there before. Was it intended that this exception was ignored just with a print?

Reply:
I think this line returning is the one causing the self tests to fail
Still don't understand why we would silently ignore it.
One way I can fix is to catch the new InvalidResultAuxPath exception and return 1 and leave the CIJError as the back stop that will silently ignore the error.
What do you think?

Reply:
Think the auto-test are happy now.